devops/subnet-addresses/main.tf

```diff
@@ -0,0 +1,43 @@
+module "subnet_addresses" {
+  source = "hashicorp/subnets/cidr"
+
+  base_cidr_block = "172.16.0.0/16"
+  networks = [
+    {
+      name     = "eu-central-1a-private"
```

Comment:
Never hardcode region so code can be reused for other regions.
For this case no need to specify region as well, just leave "1a-private" for example.

Also in order to properly segment/calculate network you need to start with bigger networks then you gradually move to smaller ones in this array.